bolt-design-system/bolt
# Fix bolt-text component spacing in IE

## Jira

http://vjira2:8080/browse/BDS-1501

## Summary

CSS changes to the bolt-text component caused a regression in IE. Bolt-text elements aren't getting any of the `display` or `margin` styles. Update the CSS to fix this.

## Details

We switched to using `%extends` in our SCSS ([see code](https://github.com/bolt-design-system/bolt/commit/5b93b8eeb705be7d7833d776f926a07f6bb19c2e#diff-9897f76a3cc1b33131c566637af20038R7)) but that automatically groups selectors together, which breaks the `:host` selector needed for IE. Manually separate out the CSS rules to fix.

## How to test

- Review code changes
- Run locally. View any `bolt-text` component demo page in IE to verify `bolt-text` is getting proper bottom margin. See example below.

**Broken:**

![image](https://user-images.githubusercontent.com/36336/60137891-5e90e500-9776-11e9-8f0a-6c57d466f6d4.png)

**Fixed:**

![image](https://user-images.githubusercontent.com/36336/60137851-40c38000-9776-11e9-83f8-3609db75468e.png)
